pandas/_libs/tslibs/conversion.pyx

```diff
@@ -48,13 +91,227 @@ cdef class _TSObject:
             return self.value
 
 
+# helper to extract datetime and int64 from several different possibilities
```

Comment:
so I would rather you integrate the comments with the doc-strings; in this case your comment is not useful as the doc-string is pretty good

Reply:
No big deal, but the comment was cut/pasted from the original.  Will change.